WalletWasabi.Gui/Tabs/WalletManager/LoadWalletView.xaml

```diff
@@ -63,8 +63,8 @@
               <controls:Spinner Height="15" Width="15" Margin="0 0 8 0" />
               <TextBlock Text="Looking for hardware wallets..." FontWeight="Bold" />
             </StackPanel>
-            <TextBlock Text="Make sure you set up and logged into the device, usually with a PIN. Some hardware wallets are picky about their USB cables, so you may want to try out multiple ones." TextWrapping="Wrap" />
```

Comment:
How about
`Make sure you have initialized the hardware wallet and log into it with your PIN.`

Reply:
Please read the original sentence. That's accurate. Then see how your sentence changes the meaning to an inaccurate one 😄

Reply:
Shouldn't we say `Make sure you ARE set up ...`? if not then I don't see the difference between the original sentence and my proposition.

Reply:
No consensus here. Use the original sentence. Other fixes are fine.

Reply:
I still don't see the difference between these two sentences
`Make sure you set up and logged into the device`
`Set up the device and log in to it`

How about
`Be sure to set up and log into the device`

I would love to edit the next part as follows (and also put it on a new line):
`Some hardware wallets are picky about their USB cables, so you may want to try out multiple ones.`
`Hardware wallets are picky about their USB cables, so try out multiple ones.`

Reply:
Fix the PR with your final version and try to make the last proposal.

Reply:
Please take a look now.